components/ApiUserDashboard.js

```diff
@@ -0,0 +1,71 @@
+import { Component } from 'react'
+import { Alert, Button } from 'react-bootstrap'
+
+import ApiKeyList from './ApiKeyList'
+import RequestLogsDashboard from './RequestLogsDashboard'
+import {OTP_PLAN_URL} from '../util/constants'
+
+/**
+ * The high-level component visible to an ApiUser when they log in. This
+ * describes the user's API keys, points to some docs, and shows API key usage.
+ */
+class ApiUserDashboard extends Component {
+  render () {
+    const {apiUser, clearWelcome, showWelcome} = this.props
+    // Sample OTP URL to show in sample CURL command
+    const samplePlanUrl = `${OTP_PLAN_URL}?fromPlace=P%26R%3A%3A28.45119%2C-81.36818&toPlace=Downtownish%3A%3A28.54834%2C-81.37745`
+    // Use the first API key value in the sample CURL command.
+    const sampleKey = apiUser.apiKeys[0] ? apiUser.apiKeys[0].value : null
+    return (
+      <>
+        {showWelcome &&
+          <Alert
+            variant='success'
+            onClose={clearWelcome}
+            dismissible>
+            <Alert.Heading>API User Created!</Alert.Heading>
+            <p>
+              Welcome to your new API User account. To get started, check out
+              your API keys below.
+            </p>
+          </Alert>
+        }
+        <h3>{apiUser.appName} API Keys</h3>
+        <Alert variant='info'>
+          Note: copy/paste the API key values below for use in
+          the <code>x-api-key</code> header when making any request to the RMCE
+          API.
+        </Alert>
+        <ApiKeyList apiUser={apiUser} />
+        <h3>Interacting with the RMCE API</h3>
+        {sampleKey
+          ? <div>
+            <p>
+              <strong>Tip:</strong>{' '}
+              Try planning a trip with the following CURL command:
+            </p>
+            <pre>
+              {`curl -H 'x-api-key: ${sampleKey}' '${samplePlanUrl}'`}
+            </pre>
+          </div>
+          : null
+        }
+        <p>
+          <Button
+            variant='primary'
+            size='lg'
+            // FIXME: Why is this env variable undefined?
```

Comment:
Or cache the variable in `constants.js`. That's probably the easiest workaround.

Reply:
I don't think that's the issue. I think I resolved the issue in: https://github.com/ibi-group/otp-admin-ui/pull/14/files#diff-5d0c276360a637d1b787a57760665fbeR28. 

But there probably needs to be a separate PR (from either of these) to fix how environment variables are handled because I think next/vercel has changed this a bit recently: https://vercel.com/docs/configuration#introduction/configuration-reference